packages/linux-driver-addons/dvb/hauppauge/patches/amlogic/driver.dvb.hauppauge-01-add-AML-specific-to-backports.patch

```diff
@@ -1,12 +1,14 @@
 --- a/backports/backports.txt
 +++ b/backports/backports.txt
-@@ -21,6 +21,9 @@
+@@ -21,6 +21,11 @@
  
  # All supported versions need those patches
  [9.255.255]
 +add linux-301-AML-videobuf-resource.patch
 +add linux-302-AML-amlogic-video-dev.patch
 +add linux-303-AML-meson-ir.patch
++add linux-305-AML-dmx_h.patch
++add linux-306-AML-build-dvb-avl.patch
```

Comment:
This causes a breakage for other Amlogic based devices as the patch attempts to patch a file that doesn't exist, this patch should only be included if avl6862-aml is built.

Reply:
The issue was only in community build. Build  dvb-aml  is allow only for devices with avl6862-aml driver in :
https://github.com/LibreELEC/LibreELEC.tv/pull/2451/files/66acab5c953efec53bdd30c2d348b4c1fde3c1eb..bbfc05e912bc555a9f33faec310d7bbb02cc24b4#diff-c139070658fb360e28224a057c75b3a5

Reply:
The issue was only in community build with different patches. Look in next comment.